src/serialization/deserialize-assets.js

```diff
@@ -22,37 +22,29 @@ const deserializeSound = function (sound, runtime, zip, assetFileName) {
         return Promise.resolve(null);
     }
 
-    const assetId = sound.assetId;
-
-    // TODO Is there a faster way to check that this asset
-    // has already been initialized?
-    if (storage.get(assetId)) {
-        // This sound has already been cached.
-        return Promise.resolve(null);
-    }
```

Comment:
We'll have to see if this affects project load performance very much — previously we would only need to get a specific asset once. But now we will be relying on the browser cache to return this quickly.

Reply:
We should probably test this on projects with lots of assets and on lots of devices...